dev-python/shiboken/shiboken-5.12.3.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+CMAKE_MIN_VERSION=3.1.0
+
+inherit cmake-utils llvm python-r1
+
+MY_P="pyside-setup-everywhere-src-${PV}"
+
+DESCRIPTION="Python binding generator for C++ libraries"
+HOMEPAGE="https://wiki.qt.io/PySide2"
+SRC_URI="https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-${PV}-src/${MY_P}.tar.xz"
+
+# The "sources/shiboken2/libshiboken" directory is triple-licensed under the
+# GPL v2, v3+, and LGPL v3. All remaining files are licensed under the GPL v3
+# with version 1.0 of a Qt-specific exception enabling shiboken2 output to be
+# arbitrarily relicensed. (TODO)
+LICENSE="|| ( GPL-2 GPL-3+ LGPL-3 ) GPL-3"
+SLOT="2"
+KEYWORDS="~amd64 ~x86"
```

Comment:
**Great question(s).** I have, of course, tested nothing. Since PySide2 officially publishes [32-bit Python 3 Windows wheels](https://bugreports.qt.io/browse/PYSIDE-646?jql=project%20%3D%20PYSIDE%20AND%20text%20~%2032-bit), my working assumption is that the same codebase *should* theoretically support 32-bit Linux environments. As counter-evidence, the official Qt for Python wiki lists [32-bit as officially unsupported under Linux](https://wiki.qt.io/Qt_for_Python).

Maybe it does? Maybe it doesn't. I'm happy to leave this as is for the moment and farm out testing to our intrepid userbase. Of course, I'm also happy to remove the `~x86` and be done with it. I defer to your magnanimous wisdom here.

Reply:
Then drop the keyword. x86 is becoming less and less common and is no longer considered a "default" arch for ebuilds.

Reply:
**Gotcha.** You're right. Anyone still on `x86` has bigger problems than lack of PySide2 support. :fire: